### PR TITLE
Update OS X to Mac

### PR DESCRIPTION
**Pull request by torn4dom4n**

### CODE REVIEW

File: docs/languages/cpp.md [hunk 1]
```diff
@@ -166,7 +166,7 @@ After you have set up the basics of your debugging environment as specified in [
 VS Code supports the following debuggers for C/C++ depending on the operating system you are using:
 
 * **Linux**: GDB
-* **OS X**: LLDB or GDB
+* **Mac**: LLDB or GDB
```
Comment (gregvanl) [inline docs/languages/cpp.md]: Hi, thank you for the PR. The CPP topic is owned by the team who owns the C++ extension and not VS Code. I think they have only tested this on OS X so I don't think they can update this yet to Mac or macOS.

File: release-notes/June_2016.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ MetaSocialImage: June_2016_social.jpg
...
+[Mac](https://az764295.vo.msecnd.net/stable/e6b4afa53e9c0f54edef1673de9001e9f0f547ae/VSCode-darwin-stable.zip) | Linux 64-bit [.zip](https://az764295.vo.msecnd.net/stable/e6b4afa53e9c0f54edef1673de9001e9f0f547ae/VSCode-linux-x64-stable.zip) [.deb](https://az764295.vo.msecnd.net/stable/e6b4afa53e9c0f54edef1673de9001e9f0f547ae/code_1.3.1-1468329898_amd64.deb) [.rpm](https://az764295.vo.msecnd.net/stable/e6b4afa53e9c0f54edef1673de9001e9f0f547ae/code-1.3.1-1468329898.el7.x86_64.rpm) | Linux 32-bit [.zip](https://az764295.vo.msecnd.net/stable/e6b4afa53e9c0f54edef1673de9001e9f0f547ae/VSCode-linux-ia32-stable.zip) [.deb](https://az764295.vo.msecnd.net/stable/e6b4afa53e9c0f54edef1673de9001e9f0f547ae/code_1.3.1-1468330706_i386.deb) [.rpm](https://az764295.vo.msecnd.net/stable/e6b4afa53e9c0f54edef1673de9001e9f0f547ae/code-1.3.1-1468330706.el7.i386.rpm)
```
Comment (torn4dom4n) [inline release-notes/June_2016.md]: Oh. I will change Mac to OS X in June_2016.md and cpp.md.

Comment (torn4dom4n) [inline release-notes/June_2016.md]: Do you want update OS X to Mac in v1_5.cpp, @gregvanl?

Comment (gregvanl) [inline release-notes/June_2016.md]: Good idea since it is still the active release. Thanks.

**Comment by torn4dom4n** [comment]
I changed Mac to OS X.

**Comment by gregvanl** [comment]
Thanks!

**Pull request merged**